src/MerQure.RbMQ/Config/RabbitMqConfigurationSection.cs

```diff
@@ -1,13 +1,8 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace MerQure.RbMQ.Config
 {
-    public class RabbitMqConfigurationSection : ConfigurationSection
+    public class RabbitMqConfigurationSection 
     {
         private const string DurablePropertyName = "durable";
         private const string AutoDeleteQueuePropertyName = "autoDeleteQueue";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'AutoDeleteQueuePropertyName'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1144)